### ISSUE TITLE
Wings couldn't generate any executable workflow for the template

### ISSUE THREAD

**Issue by mosoriob** [KnowledgeCaptureAndDiscovery/wings#147]
**To Reproduce**
1. Clone repository
2. `docker-compose up -d`
3. http://localhost:8080/wings-portal/
4. Upload domain: https://www.wings-workflows.org/domains/students/CaesarCypher.zip
5. Plan workflow

**Screenshots**

![image](https://user-images.githubusercontent.com/2365990/210384127-ac05eb5d-a6f7-41f6-8572-55ad920beb48.png)

**Comment by mosoriob** [comment]
Version 5.1.3 works.

**Comment by mosoriob** [comment]
No. Version 5.1.3 is affected by a similar issue

```java
wings_1  | org.apache.jena.datatypes.DatatypeFormatException: Lexical form '[1]' is not a legal instance of Datatype[http://www.w3.org/2001/XMLSchema#integer -> class java.math.BigInteger] Lexical form '[1]' is not a legal instance of Datatype[http://www.w3.org/2001/XMLSchema#integer -> class java.math.BigInteger] during parse -org.apache.xerces.impl.dv.InvalidDatatypeValueException: cvc-datatype-valid.1.2.1: '[1]' is not a valid value for 'integer'.
```

**Comment by mosoriob** [comment]
Edit a component doesn't work 

![image](https://user-images.githubusercontent.com/2365990/210386402-c003f603-4adf-4c51-8ce9-98d078eb7d4c.png)

**Comment by varunratnakar** [comment]
Should be fixed at 5.3.4

**Comment by varunratnakar** [comment]
Verified fix
